test-plan.md

```diff
@@ -106,6 +106,9 @@ To setup a test project to verify on you can do:
 * Completion is available for types that exist in the project (i.e. `Program`)
 * Typing `@model DateTime` prompts for completion for the `model` symbol and the `DateTime` symbol.
 
+#### C# Lightbulbs
+* Typing `@{ var x = new HtmlString("sdf"); }`results in squiggles under `HtmlString`. Selecting `HtmlString` and pressing Ctrl+. results in a change suggestion to fully-qualify that type name as `Microsoft.AspNetCore.Html.HtmlString`. Hitting enter on that suggestion resolves the squiggles and changes the line.
```

Comment:
You may want to be a little more specific here on which entry to select to future proof ourselves for when we add more light bulbs

Reply:
I feel like I was super specific, `Microsoft.AspNetCore.Html.HtmlString` is the entire text which is shown by that light bulb. Can you think of a way I could make that wording clearer?

Reply:
ya this is why you don't leave comments later at night. You were totally clear enough lol.